src/community/people/components/organisms/AddNewResourceFlow/PersonalDetailsSection/GeneralDetailsSection.tsx

```diff
@@ -102,6 +110,10 @@ const GeneralDetailsSection = forwardRef<FormMethods, Props>(
       [employeeDataChanges]
     );
 
+    const usedStoragePercentage = useMemo(() => {
```

Comment:
check te mode and disable for enterprise @shakyaatrootcode

Reply:
This code is to get the used storage percentage and show case the banner in the UI. Did this as a fix for a demo. 

Will have to ask @adeepabandara101 whether it is not needed in the enterprise.